packages/react-instantsearch-hooks/package.json

```diff
@@ -49,7 +49,7 @@
     "@babel/runtime": "^7.1.2",
     "algoliasearch-helper": "^3.7.0",
     "dequal": "^2.0.0",
-    "instantsearch.js": "^4.39.0"
+    "instantsearch.js": "https://pkg.csb.dev/algolia/instantsearch.js/commit/70fb1c6d/instantsearch.js"
```

Comment:
This will be updated once https://github.com/algolia/instantsearch.js/pull/5024 is released.